### ISSUE TITLE
Colour calibration crashes

### ISSUE THREAD

**Issue by patriot1889** [hyperion-project/hyperion.ng#1094]
- [x] I confirm that this is an issue rather than a question.

## Bug report

Colour calibration crashes when trying to change the colours. I was originally on release 8 on a stock pi image, reinstalled the whole system with HyperBian to test, still happens.

#### Steps to reproduce

Use colour calibration tool

#### What is expected?

For it work and not crash

#### What is actually happening?

It's crashing

**LOG**
Nov 14 21:52:16 HyperBian hyperiond[858]: 2020-11-14T21:52:16.368 hyperiond HYPERION     : <ERROR> setInput() used without registerInput() for priority '254', probably the priority reached timeout

Nov 14 21:52:16 HyperBian [858]: setInput() used without registerInput() for priority '254', probably the priority reached timeout

Nov 14 21:53:16 HyperBian hyperiond[858]: terminate called after throwing an instance of 'std::out_of_range'

Nov 14 21:53:16 HyperBian hyperiond[858]:   what():  vector::_M_range_check: __n (which is 96) >= this->size() (which is 96)

Nov 14 21:53:16 HyperBian hyperiond[858]: Hyperion caught signal :SIGABRT

Nov 14 21:53:16 HyperBian hyperiond[858]: 2020-11-14T21:53:16.719 hyperiond CORE         : <ERROR>         /lib/arm-linux-gnueabihf/libc.so.6(__default_rt_sa_restorer+0) [0x7562b130]

Nov 14 21:53:16 HyperBian [858]:         /lib/arm-linux-gnueabihf/libc.so.6(__default_rt_sa_restorer+0) [0x7562b130]

Nov 14 21:53:16 HyperBian systemd[1]: hyperion.service: Main process exited, code=killed, status=6/ABRT

Nov 14 21:53:16 HyperBian systemd[1]: hyperion.service: Failed with result 'signal'.

#### System

Hyperion Server: 
- Build:       (HEAD detached at 08909b8a) (Paulchen-Panther-75963ca9/08909b8a-1600115237)
- Build time:  Sep 14 2020 21:05:54
- Git Remote:  https://github.com/hyperion-project/hyperion.ng
- Version:     2.0.0-alpha.8
- UI Lang:     auto (BrowserLang: en-gb)
- UI Access:   expert
- Avail Capt:  dispmanx,v4l2,framebuffer,qt

Hyperion Server OS: 
- Distribution: HyperBian GNU/Linux 10 (buster)
- Arch:         arm
- Kernel:       linux (5.4.51-v7+ (WS: 32))
- Browser:      Mozilla/5.0 (Macintosh; Intel Mac OS X 10_14_6) AppleWebKit/605.1.15 (KHTML, like Gecko) Version/14.0 Safari/605.1.15

**Comment by danielbayley80** [comment]
Im seeing the same,
 HyperBian 
 Pi 3 A+
 
 requires me to force it on and off to recover.

**Comment by patriot1889** [comment]
I have tried out the project "HyperHDR", as it claims to have a lower latency and capable of using more of the pi processing power due to being multithreaded.

Anyway, HyperHDR also had this issue as it is based on Hyperion.ng - until the newest release. v15, from around 23 hours ago fixes this problem completely! It might be worth taking a look to see what the dev has changed?

"Fixes: better instances interaction and many of discovered bugs were fixed including critical ones like heap corruption while picking up a color during calibration."

https://github.com/awawa-dev/HyperHDR
